src/OSPSuite.UI/Views/Importer/CsvSeparatorSelectorView.cs

```diff
@@ -1,51 +1,87 @@
-﻿using System;
-using System.Collections.Generic;
-using OSPSuite.Assets;
+﻿using System.Collections.Generic;
+using DevExpress.XtraEditors;
+using DevExpress.XtraLayout.Utils;
+using OSPSuite.Presentation.Extensions;
 using OSPSuite.Presentation.Presenters.Importer;
 using OSPSuite.Presentation.Views.Importer;
+using static OSPSuite.Assets.Captions.Importer;
 
 namespace OSPSuite.UI.Views.Importer
 {
-   public partial class CsvSeparatorSelectorView : BaseModalView, ICsvSeparatorSelectorView
+   public partial class CSVSeparatorSelectorView : BaseModalView, ICSVSeparatorSelectorView
    {
-      private ICsvSeparatorSelectorPresenter _presenter;
-      public CsvSeparatorSelectorView()
+      private ICSVSeparatorSelectorPresenter _presenter;
+      private const char Comma = ',';
+      private const char Period = '.';
+      private readonly List<char> _columnSeparatorList = new List<char> { Comma, ' ', ';', Period };
+      private readonly List<char> _decimalSeparatorList = new List<char> { Period, Comma };
+
+      public CSVSeparatorSelectorView()
       {
          InitializeComponent();
          fillSeparatorComboBox();
-         separatorComboBoxEdit.EditValueChanged += (s, a) => OnEvent(onSeparatorChanged);
+         columnSeparatorComboBoxEdit.EditValueChanged += (s, a) => OnEvent(onSeparatorChanged);
+         decimalSeparatorComboBoxEdit.EditValueChanged += (s, a) => OnEvent(onSeparatorChanged);
       }
 
       public override void InitializeResources()
       {
          base.InitializeResources();
-         separatorDescriptionLabelControl.Appearance.TextOptions.WordWrap = DevExpress.Utils.WordWrap.Wrap;
+         previewMemoEdit.Enabled = false;
+         previewMemoEdit.Properties.LinesCount = _presenter.PreviewLineCount;
+         previewMemoEdit.Properties.AutoHeight = true;
+         Caption = SeparatorSelection;
+
+         columnSeparatorLayoutControlItem.Text = ColumnSeparator.FormatForLabel();
+         decimalSeparatorLayoutControlItem.Text = DecimalSeparator.FormatForLabel();
       }
 
       private void fillSeparatorComboBox()
       {
-         var separatorList = new List<char>() {',', '.', ' ', ';'};
-
-         foreach (var separator in separatorList)
-         {
-            separatorComboBoxEdit.Properties.Items.Add(separator);
-         }
+         columnSeparatorComboBoxEdit.Properties.Items.AddRange(_columnSeparatorList);
```

Comment:
This fixes the issue so I am only going to put it as a suggestion for another time. We should have a DTO, a binding and not do all this settings, EditValueChanged etc by hand. Use the framework. 
/cc @georgeDaskalakis Not sure if you implemented this. Much better to use the screenbinder for these use cases